microsoft-edge/toc.yml

```diff
@@ -1434,11 +1434,25 @@
             - name: Publish a UWP WebView2 app to the Microsoft Store
               href: webview2/how-to/publish-uwp-app-store.md
               displayName: Partner Center, Microsoft Store
+# -----------------------------------------------------------------
+# relnotes
+        - name: Release Notes
```

Comment:
![image](https://github.com/MicrosoftDocs/edge-developer/assets/5773562/c76dfa42-256a-4c99-8b25-943e53696cd3)
I think the way that the TOC hierarchy right now is a little verbose (the words `for the WebView2 SDK`) is repeated.

Was wondering if we could make the hierarchy more like:
Release Notes for the WebView2 SDK
    About
    Release Notes
    Archived Release Notes

Reply:
it's easy to shorten TOC titles as long as I make sure to define the "displayName:" field to contain the full title, so that the "Filter by title" upper left UI gives the expected results when entering the full title.  eg:
 
- name: WinUI 3 (Windows App SDK)
  href: webview2/get-started/winui.md
  displayName: Get started with WebView2 in WinUI 3 (Windows App SDK) apps  # top-of-page title
 
Rules for shortening title in TOC:
Must use same words (a subset) and order as in the full title.
Must define "displayName:" including full article title.

Reply:
shortened